ui/articles-aem-renderer/src/main/java/org/cru/godtools/articles/aem/db/ResourceDao.kt

```diff
@@ -1,14 +1,13 @@
 package org.cru.godtools.articles.aem.db
 
-import android.arch.lifecycle.LiveData
 import android.arch.persistence.room.Dao
 import android.arch.persistence.room.Insert
 import android.arch.persistence.room.OnConflictStrategy
 import android.arch.persistence.room.Query
 import android.net.Uri
 import android.support.annotation.WorkerThread
 import org.cru.godtools.articles.aem.model.Resource
-import java.util.Date
+import java.util.*
```

Comment:
I had to change my code style settings within Android Studio to prevent the code formatter from doing wildcard imports.

Reply:
ktlint will complain about wildcard imports.

Using a wildcard import can lead to unexpected results when new code uses an unexpected class that was included via a wildcard.